spec/models/spree/order_spec.rb

```diff
@@ -89,5 +89,17 @@
         end
       end
     end
+
+    describe 'destruction' do
+      let!(:order) { create :invoiceable_order }
+
+      context "when destroyed" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.